src/index.ts

```diff
@@ -1,67 +1,67 @@
 import * as faker from 'faker';
 import { mapValues } from 'lodash';
 
-type SequenceFunction = (counter: number) => unknown;
+export type SequenceFunction = (counter: number) => unknown;
 
-interface SequenceGenerator {
+export interface SequenceGenerator {
   generatorType: 'sequence';
   userProvidedFunction: SequenceFunction;
   call: (userProvidedFunction: SequenceFunction, counter: number) => unknown;
 }
 
-interface FakerGenerator {
+export interface FakerGenerator {
   generatorType: 'faker';
   call: (fake: Faker.FakerStatic) => any;
 }
 
-interface PerBuildGenerator {
+export interface PerBuildGenerator {
   generatorType: 'perBuild';
   func: () => any;
   call: (f: () => any) => any;
 }
 
-interface OneOfGenerator {
+export interface OneOfGenerator {
   generatorType: 'oneOf';
   options: any[];
   call: <T>(options: T[]) => T;
 }
 
-type FieldGenerator =
+export type FieldGenerator =
   | FakerGenerator
   | SequenceGenerator
   | OneOfGenerator
   | PerBuildGenerator;
 
-type Field =
+export type Field =
   | string
   | number
   | null
   | FieldGenerator
   | { [x: string]: Field | {} }
   | any[];
 
-type FieldsConfiguration<FactoryResultType> = {
+export type FieldsConfiguration<FactoryResultType> = {
   readonly [x in keyof FactoryResultType]: Field;
 };
 
-interface Overrides<FactoryResultType> {
```

Comment:
The generic wasn't used within the `Overrides` interface, so we can remove it.

...unless you had a reason for keeping it?